### DISCUSSION TITLE
corte

### DISCUSSION THREAD

**Discussion by fabio750** [arkypita/LaserGRBL#1855] [General]
hola, consulta si alguien me puede ayudar
quiero cortar una imagen que diseñe en ilustraitor, y la maquina me lo corta dos veces, el vectorizado (grbl) me toma dos lineas, no se cual es el problema, gracias.

**Comment by Ams-Laser** [comment]
Fabio, el problema es el grosor de la línea, haslo lo más delgado que
permita y funciona
AMS-Laser

**Reply by fabio750** [reply]
si ya he probado al minimo 0,25 pt, y no funciona, la verdad ya no se que hacer!, gracias lo mismo por tu tiempo

**Reply by Tribo-2dot0** [reply]
What format do you save the file in?
If you export as a bitmap and try to vectorize in LaserGrbl, you will have a cut line on each side of the original drawing line... instead save as .svg and you will have a single path.
or try using Centerline in LaserGrbl.

**Reply by fabio750** [reply]
siempre  guarde en jpg y png, y no funciona, probe centerline y tampoco funciona, y probe como me dijo en formato SVG y el programa de grbl no lo reconoce , no funciona bien, estoy desconcertado,muchas gracias igual por su  tiempo.

**Reply by Tribo-2dot0** [reply]
Mostre aqui a imagem que está tentando fazer.

**Reply by fabio750** [reply]
![Sin título-8](https://user-images.githubusercontent.com/108538838/178067207-39996182-06bd-4be6-990c-bce42c3cd5f6.png)

**Reply by fabio750** [reply]
esta realizado con lineas finas, pero igual el programa ve el acho de la linea y corta el exterior y el interior de cada linea.
Lo mas raro que hay algunas figuras que si la pudes cortar, pero esta imagen pasa dos veces por cada linea, siempre en PNJ porque SVG no funciona, gracias.

**Reply by Tribo-2dot0** [reply]
Quando você vetoriza uma linha (como as da imagem que você postou), a vetorização considera a linha como uma área e fará os dois lados dessa área, mesmo que você considere isso uma linha (da para ver os dois lados da linha). Se quiser que seja realmente uma única linha, tens que fazer esse desenho diretamente no  programa de vetor (com o inkscape) usando a ferramenta de círculo, daí terá realmente uma única linha. Apartir de uma imagem bitmap a única maneira de conseguir uma única linha é usando a vetorização Centerline (que não fica tão bom quanto fazer diretamente com Inkscape ou Corel Draw)

**Reply by fabio750** [reply]
Buen dato!!!! yo uso adobe, pero voy a probar con inkscape, muchas gracias por tu atencion y tiempo!!!!
Saludos Fabio.
